rkyv/src/collections/swiss_table/map.rs

```diff
@@ -419,3 +454,43 @@ impl<K, V, H> ExactSizeIterator for ValuesMut<'_, K, V, H> {
 }
 
 impl<K, V, H> FusedIterator for ValuesMut<'_, K, V, H> {}
+
+/// A builder for accessing entries in an [`ArchivedHashMap`] with a custom hash and comparison function.
+///
+/// This is a lower-level API similar to `hashbrown::HashMap::raw_entry()`.
+pub struct RawEntryBuilder<'a, K, V, H> {
```

Comment:
This is a large API surface, and rkyv's goal is to match (as closely as possible) the API for items in `std`, not `hashbrown`.